pkg/config/registry/version_override.go

```diff
@@ -4,35 +4,39 @@ import (
 	"strings"
 
 	"github.com/aquaproj/aqua/v2/pkg/expr"
+	"github.com/sirupsen/logrus"
+	"github.com/suzuki-shunsuke/logrus-error/logerr"
 )
 
-func (pkgInfo *PackageInfo) setTopVersion(v string) (*PackageInfo, error) {
+func (pkgInfo *PackageInfo) setTopVersion(logE *logrus.Entry, v string) *PackageInfo {
 	sv := v
 	if pkgInfo.VersionPrefix != nil {
 		prefix := *pkgInfo.VersionPrefix
 		if !strings.HasPrefix(v, prefix) {
-			return nil, nil //nolint:nilnil
+			return nil
 		}
 		sv = strings.TrimPrefix(v, prefix)
 	}
 	a, err := expr.EvaluateVersionConstraints(pkgInfo.VersionConstraints, v, sv)
 	if err != nil {
-		return nil, err //nolint:wrapcheck
+		logerr.WithError(logE, err).Debug("evaluate the version_constraint")
```

Comment:
If it fails to evaluate `version_constraint`, output debug log and the evaluation result is treated as `false`.